frontend/public/components/notification-drawer.tsx

```diff
@@ -105,25 +108,48 @@ const getAlertNotificationEntries = (
     : [];
 
 const getUpdateNotificationEntries = (
-  isLoaded: boolean,
-  updateData: ClusterUpdate[],
+  cv: ClusterVersionKind,
   isEditable: boolean,
   toggleNotificationDrawer: () => void,
-): React.ReactNode[] =>
-  isLoaded && !_.isEmpty(updateData) && isEditable
+): React.ReactNode[] => {
+  const updateData: ClusterUpdate[] = getSortedUpdates(cv);
+  const currentChannel = cv?.spec?.channel;
+  const currentPrefix = splitClusterVersionChannel(currentChannel)?.prefix;
+  const similarChannels = getSimilarClusterVersionChannels(currentPrefix);
+  const newerChannel = getNewerClusterVersionChannel(similarChannels, currentChannel);
+  const newerChannelVersion = splitClusterVersionChannel(newerChannel)?.version;
+  return isEditable
     ? [
-        <NotificationEntry
-          actionPath="/settings/cluster"
-          actionText="Update cluster"
-          key="cluster-udpate"
-          description={updateData[0].version || 'Unknown'}
-          type={NotificationTypes.update}
-          title="Cluster update available"
-          toggleNotificationDrawer={toggleNotificationDrawer}
-          targetPath="/settings/cluster"
-        />,
+        !_.isEmpty(updateData) && (
```

Comment:
This puts `false` in the array when there are no updates, which I'd think would cause problems. Similar problem below when `newerChannel` isn't defined.